vault/resource_password_policy_test.go

```diff
@@ -0,0 +1,58 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
+func TestAccPasswordPolicy(t *testing.T) {
+
+	policyName := acctest.RandomWithPrefix("test-policy")
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testProviders,
+		CheckDestroy: testAccPasswordPolicyCheckDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccPasswordPolicy(policyName, "length = 20\nrule \"charset\" {\n  charset = \"abcde\"\n}\n"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("vault_password_policy.test", "name", policyName),
+					resource.TestCheckResourceAttrSet("vault_password_policy.test", "policy"),
+				),
+			},
+		},
+	})
+}
+
```

Comment:
There needs to be a test for the import functionality. Can you add one?

Reply:
Adding one now